### DISCUSSION TITLE
Sonner Component

### DISCUSSION THREAD

**Discussion by charmbyte** [mantinedev/mantine#5824] [Feature requests]
Just out of curiosity - are there any plans to add a Sonner component?

**Comment by rtivital** [comment]
What is Sooner component?

**Comment by charmbyte** [comment]
Sorry, it's a newer component that I've seen on the UI "market". It's similar to a Toast / Notification but is a little more stylized and condensed (useful for smaller screens).

Here is the shadcn example: https://ui.shadcn.com/docs/components/sonner

EDIT: typo

**Reply by rtivital** [reply]
Mantine already provides `@mantine/notifications` component. It is not planned to add another components/packages to manage notifications/toasts.

**Reply by charmbyte** [reply]
Understandable. Thank you.
